lotr_homework/js/app.js

```diff
@@ -71,17 +94,26 @@ const makeMiddleEarth = () => {
 const makeHobbits = () => {
 
   console.log('Make hobbits');
-
+  const $hobbitUl = $('<ul/>');
+  $('#The-Shire').append($hobbitUl);
   // 1. display an unordered list of the hobbits in the shire.
 
   // 2. give each hobbit a class of "hobbit"
+  for( let j = 0; j < hobbits.length; i++); {
+    let $hobbitLi = $('<li/>');
+    $hobbitLi.text(hobbits[i]);
```

Comment:
this is causing an error.  you used `j` above.  not sure how you did the rest of this with this error in here, as all problems depend on previous problems working...